fiscalsim_us/tests/policy/baseline/gov/states/mn/tax/income/mn_amt_taxable_income.yaml

```diff
@@ -7,7 +7,7 @@
     us_govt_interest: 3_000
     state_code: MN
   output:
-    mn_amt_taxable_income: 200_000 - 1_000 - 3_000
+    mn_amt_taxable_income: 117097.5

```

Comment:
@mitchellpound.
1. We should round these to the nearest dollar. You can do this in the variable `.py` files.
2. These amounts are significantly different from their original values. Can you explain why?

Reply:
@rickecon, 
- I added a rounding rule to the `mn_amt.py` file, do you think we should a rule to the `mn_amt_taxable_income.py` file as well? 
- The MN amt taxable income is not computed with just itemized or the standard deduction. The standard deduction is always allowed, even if itemizing. So even if the tax payer decides to itemize and deduct the interest and charitable deductions they will still take the (partially phased out since the income is over the threshold) standard deduction, so the amt taxable income will be much less than just $200,000 - 1,000 - 3,000$